composer.json

```diff
@@ -11,7 +11,7 @@
         }
     ],
     "require": {
-        "php": ">=5.3.0"
+        "php": "^7.2 || ^8.0"
```

Comment:
As noted on #95, I think it makes sense to allow v7 here, due to the large install base of RHEL 7, Ubuntu 20.04, and Debian 11, all of which pin to PHP 7.4. We will have PHP 7 users for some years, even if it's not technically in support, and there's no reason to restrict its usage artificially.

Reply:
Aren't there some places where you wanted to use `mixed`, though?  Why punt on that unnecessarily?

Most of the other PSR updates have targeted 8.x (they used `mixed` or union types), and in practice... how many people use the pinned old versions of PHP on distros rather than PPCs?  I admittedly only know from one corporate environment (my own), but we're ignoring that.  I don't get the sense that distro versions matters anywhere near as much as it used to.

Reply:
> Aren't there some places where you wanted to use `mixed`, though? Why punt on that unnecessarily?

We can do a 2.1 release that restricts to 8.0 and up and adds `mixed`.

Reply:
> I don't get the sense that distro versions matters anywhere near as much as it used to.

They absolutely do.

We (Zend) just released our annual PHP Landscape Report. It is _very_ representative of the broader PHP community, has a representative mix of developers to decision makers, and covers a broad spectrum of company sizes.

This year, as in all years past that I can remember, 60% of respondents are _currently_ using EOL versions of PHP.

There's no reason to artificially restrict what features or security fixes they can get in the PHP libraries they consume because they are on an EOL version of PHP. They may be getting security patches for the runtime already (e.g., via an operating system, or via a commercial LTS provider), so limiting what they can use makes it harder to keep the libraries they use updated.